src/java.base/share/classes/com/sun/crypto/provider/HmacCore.java

```diff
@@ -102,6 +112,12 @@ abstract class HmacCore extends MacSpi implements Cloneable {
         this.k_ipad = new byte[blockLen];
         this.k_opad = new byte[blockLen];
         first = true;
+
+        /*[IF CRIU_SUPPORT]*/
+        if (InternalCRIUSupport.isCheckpointAllowed()) {
```

Comment:
`InternalCRIUSupport` needs to be imported.

Reply:
Updated

Reply:
Updated